src/ui/components/FeatureConfigPage.tsx

```diff
@@ -90,6 +129,54 @@ const FeatureConfigPage: FC = () => {
           Enroll
         </Button>
       </Form>
+
+      <Modal show={!!enrollError} onHide={handleModalClose}>
+        <Modal.Header closeButton>
+          <Modal.Title>Force Enrollment</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          {enrollError && (
+            <>
+              {enrollError.activeEnrollment &&
+                enrollError.slugExistsInStore && (
+                  <p>
+                    There already is an enrollment for the feature:{" "}
+                    <strong>{`nimbus-devtools-${selectedFeatureId}-${isRollout ? "rollout" : "experiment"}`}</strong>
```

Comment:
Can we pull out `const slug = `nimbus-devtools-${selectedFeatureId}-${isRollout ? "rollout" : "experiment"}` somewhere above ? Ideally we wouldnt be defining this in two places, but thats a later problem.